packages/conf-libmosquitto/conf-libmosquitto.1/opam

```diff
@@ -3,9 +3,13 @@ maintainer: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
 authors: [ "Markus W. Weissmann <markus.weissmann@in.tum.de>" ]
 homepage: "https://mosquitto.org/"
 license: [ "Eclipse Public License v1.0" "Eclipse Distribution License 1.0" ]
-build: [["ls" "/usr/include/mosquitto.h"]]
+build: [
+  ["ls" "/usr/include/mosquitto.h" {os != "darwin"}]
+  ["ls" "/usr/local/Cellar/mosquitto/1.4.14/include/mosquitto.h" {os = "darwin"}]
```

Comment:
This is unfortunately very brittle... the latest brew already has `/usr/local/Cellar/mosquitto/1.4.14_2/` instead. I think you need a configure script of some sort...

Reply:
Thank you for following up, how's Homebrew's `brew ls mosquitto` instead?

Reply:
Fine by me, although I think you need to add `--versions` for the desired behaviour.